shard.lock

```diff
@@ -0,0 +1,6 @@
+version: 1.0
+shards:
+  lsp:
+    github: crystal-lang-tools/lsp
+    commit: 19e5f71fb64fa79c5e077e45b60802c06b807353
```

Comment:
:exclamation::exclamation:  The commit is wrong, you need do run `shards update` and commit the new `shards.lock` (the commit in the lock file should start with `acad8ce`)

Reply:
This is weird, `shards update` doesn't change the commit hash, reinstalling also leads to the commit hash being `19e5f71...`

Reply:
I ended up using `version:` instad of `tag:`.
It now works(reinstalled and checked the latest version in `lib`; it was `acad8ce..`)

Reply:
Will check tonight, but I'm pretty sure that `shards update` changed the lock file when I tried yesterday

Reply:
Please do, It seems quite weird for a tag to not register 😖